apis/Google.Cloud.Spanner.Data/docs/migration-to-2.md

```diff
@@ -0,0 +1,103 @@
+# Migration to Google.Cloud.Spanner.Data 2.0
+
+Google.Cloud.Spanner.Data version 2.0 includes a number of breaking
+changes compared with version 1.0. We expect that most users will
+not be affected by these changes, particularly when default settings
+are used. For users who are affected, this document should help you
+to migrate your application to the 2.0 version of the libraries.
+
+If you have any problems with the breaking changes and they're not
+covered in this document, please [file an
+issue](https://github.com/googleapis/google-cloud-dotnet/issues/new)
+so we can help you further.
+
+## Configuration changes
+
+- `SpannerOptions` has been removed completely. See the [configuration
+  guide](configuration.md) for more details, but for migration
+  purposes:
+  - `KeepAliveInterval` is replaced by `SessionPoolOptions.IdleSessionRefreshDelay`
+  - `MaximumPooledSessions` has no direct equivalent in 2.0; once a session
+  is part of the pool, it will be refreshed as necessary until it is evicted.
+  - `MaximumActiveSessions` is now only present in `SessionPoolOptions`
+  - `MaximumGrpcChannels` is specified in the connection string
+  - `LogLevel` is controlled on a per-logger basis; this property on
+    `SpannerOptions` was equivalent to changing
+    `Logger.DefaultLogger.LogLevel`.
+  - `PoolEvictionDelay` is now only present in `SessionPoolOptions`
+  - `ResourcesExhaustedBehavior` has moved to `SessionPoolOptions`
+  - `Timeout` is specified in the connection string
```

Comment:
and in SessionPoolOptions as well, at least according to configuration.md (unless these are two different timeouts?)

Reply:
They *are* different timeouts, and were always separate. The one that was in SpannerOptions has only moved to the connection string.

Reply:
OK